lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex

```diff
@@ -0,0 +1,84 @@
+defmodule ExVision.InstanceSegmentation.MaskRCNN_ResNet50_FPN_V2 do
+  @moduledoc """
+  An instance segmentation model with a ResNet-50-FPN backbone. Exported from torchvision.
+  """
+  use ExVision.Model.Definition.Ortex,
+    model: "maskrcnn_resnet50_fpn_v2_instance_segmentation.onnx",
+    categories: "priv/categories/coco_categories.json"
+
+  require Logger
+
+  alias ExVision.Types.BBoxWithMask
+
+  @type output_t() :: [BBoxWithMask.t()]
+
+  @impl true
+  def load(options \\ []) do
+    if Keyword.has_key?(options, :batch_size) do
+      Logger.warning(
+        "`:max_batch_size` was given, but this model can only process batch of size 1. Overriding"
+      )
+    end
+
+    options
+    |> Keyword.put(:batch_size, 1)
+    |> default_model_load()
+  end
+
+  @impl true
+  def preprocessing(img, _metdata) do
+    ExVision.Utils.resize(img, {224, 224})
+  end
+
+  @impl true
+  def postprocessing(
+        %{
+          "boxes_unsqueezed" => bboxes,
+          "labels_unsqueezed" => labels,
+          "masks_unsqueezed" => masks,
+          "scores_unsqueezed" => scores
+        },
+        metadata
+      ) do
+    categories = categories()
+
+    {h, w} = metadata.original_size
+    scale_x = w / 224
+    scale_y = h / 224
+
+    bboxes =
+      bboxes
+      |> Nx.squeeze(axes: [0])
+      |> Nx.multiply(Nx.tensor([scale_x, scale_y, scale_x, scale_y]))
+      |> Nx.round()
+      |> Nx.as_type(:s64)
+      |> Nx.to_list()
+
+    scores = scores |> Nx.squeeze(axes: [0]) |> Nx.to_list()
+    labels = labels |> Nx.squeeze(axes: [0]) |> Nx.to_list()
```

Comment:
Here the same code repeats. Maybe it's worth to move this code as functions to utils.

Reply:
@msluszniak Maybe you can add that to your PR with suggested changes

Reply:
Yes, sounds good. If there are no blockers to this PR, we can merge it and I'll rebase and add my suggestions.

Reply:
Merged, you can now rebase your PR